docs/source/user/installing.rst

```diff
@@ -262,7 +262,9 @@ information.
 +----------++--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
 | Numba     | Release date | Python                    | NumPy                      | llvmlite                     | LLVM              | TBB                         |
 +===========+==============+===========================+============================+==============================+===================+=============================+
-| 0.58.0    | 2023-09-20   | 3.8.x <= version < 3.12   | 1.21 <= version < 1.26     | 0.41.x                       | 14.x              | 2021.6 <= version           |
+| 0.58.1    | 2023-10-16   | 3.8.x <= version < 3.12   | 1.22 <= version < 1.27     | 0.41.x                       | 14.x              | 2021.6 <= version           |
```

Comment:
Release date inconsistent with release notes?

Reply:
oh yes, indeed.

Reply:
fixed in 93aea5ba6539c62e55f9f17c3ca786256b5d4b66